pulsar-common/src/main/proto/PulsarMarkers.proto

```diff
@@ -80,10 +80,12 @@ message ClusterMessageId {
 message MessageIdData {
 	required uint64 ledger_id = 1;
 	required uint64 entry_id  = 2;
+    optional int32 partition = 3 [default = -1];
 }
 
 
 /// --- Transaction marker ---
 message TxnCommitMarker {
-    required MessageIdData message_id = 1;
+    optional MessageIdData message_id = 1;
+    repeated MessageIdData messageIdList = 2;
```

Comment:
I think we can keep `repeated MessageIdData message_ids` here. Since we don't need to consider the compatibility guarantee for the first transaction release.

Reply:
Ok